pset-functions.js

```diff
@@ -31,6 +31,23 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+function blackJack(playerCardScore, dealerCardScore)  {
+
+  if (playerCardScore > 21 && dealerCardScore > 21) {
+    return 0
+  } else if (playerCardScore < 21 && dealerCardScore > 21)  {
+    return playerCardScore
+  } else if (playerCardScore > 21 && dealerCardScore < 21)  {
+    return dealerCardScore
+  } else if (playerCardScore > dealerCardScore) {
+    return playerCardScore
+  } else if (playerCardScore < dealerCardScore) {
+    return dealerCardScore
+  }
+}
+const playerCardScore = 22
+const dealerCardScore = 22
+console.log(blackJack(playerCardScore, dealerCardScore))
```

Comment:
@laraann88 Good Job on this solution here. You utilized the `blackJack` function to check all of the conditionals you created to determine the winner/loser of the game.

Take a look at this solution [here](https://github.com/jsr20190826/hw-02-functions-solution/blob/a4d789161886ff115657133c279062df8a4ca72d/pset-functions.js#L30)